tests/compile.sh

```diff
@@ -100,46 +100,42 @@ else
     CMAKE_FLAGS="${CMAKE_FLAGS} -DMULTI_GASES=OFF"
 fi
 
-if [[ "${MAKE_OPT}" == *"CCPP=Y"* ]]; then
+# FIXME - create CCPP include directory before building FMS to avoid
+# gfortran warnings of non-existent include directory (adding
+# -Wno-missing-include-dirs) to the GNU compiler flags does not work,
+# see also https://gcc.gnu.org/bugzilla/show_bug.cgi?id=55534);
+# this line can be removed once FMS becomes a pre-installed library
+mkdir -p $PATHTR/FV3/ccpp/include
+# Similar for this directory, which apparently never gets populated
+mkdir -p $PATHTR/FMS/fms2_io/include
 
-  # FIXME - create CCPP include directory before building FMS to avoid
-  # gfortran warnings of non-existent include directory (adding
-  # -Wno-missing-include-dirs) to the GNU compiler flags does not work,
-  # see also https://gcc.gnu.org/bugzilla/show_bug.cgi?id=55534);
-  # this line can be removed once FMS becomes a pre-installed library
-  mkdir -p $PATHTR/FV3/ccpp/include
-  # Similar for this directory, which apparently never gets populated
-  mkdir -p $PATHTR/FMS/fms2_io/include
+CMAKE_FLAGS="${CMAKE_FLAGS} -DCCPP=ON -DMPI=ON"
```

Comment:
Can we move all the CMAKE_FLAGS options down to build.sh? People report that  build.sh does not work any more without passing these CMAKE_FLAGS, so to avoid changes in downstream jobs, I think it's good to move the compile options down to build.sh

Reply:
Not as part of this commit.

Reply:
Only the cmake options that we know must be always ON or OFF (but then, many option that must be always ON or OFF can be removed entirely from the build system).
All other options must remain in compile.sh

Reply:
Yes, that is fine.